libs/api_common/api_common.vcxproj

```diff
@@ -129,6 +129,9 @@
       <GenerateDebugInformation>true</GenerateDebugInformation>
       <EnableUAC>false</EnableUAC>
     </Link>
+    <Lib>
```

Comment:
> Some reason these can't be added to the top-level "Directory.Build.props" file rather than duplicating them in each project file?

Reason is some don't seem to work when rooted to `Directory.Build.props` -> investigating on this with the VS Team, hence why I disabled auto-merge for the moment.